cdc_iterator.go

```diff
@@ -112,26 +129,26 @@ func (c *cdcIterator) getStartPosition() (common.CdcPosition, error) {
 		return cdcPosition, fmt.Errorf("failed to get master position: %w", err)
 	}
 
-	return common.CdcPosition{Name: masterPos.Name, Pos: masterPos.Pos}, nil
+	return common.CdcPosition{
+		ReplicationEventPosition: common.ReplicationEventPosition{
+			Name: masterPos.Name,
+			Pos:  masterPos.Pos,
+		},
+	}, nil
 }
 
 func (c *cdcIterator) Ack(context.Context, opencdc.Position) error {
 	return nil
 }
 
-func (c *cdcIterator) Read(ctx context.Context) (rec opencdc.Record, err error) {
+func (c *cdcIterator) Read(ctx context.Context) (rec opencdc.Record, _ error) {
```

Comment:
might as well remove the named variables altogether.

Reply:
fixed